enhancements/olm-curated-catalog.md

```diff
@@ -0,0 +1,353 @@
+---
+title: olm-curated-catalog
+
+authors:
+  - "@cdjohnson"
+
+reviewers:
+
+approvers:
+
+creation-date: 2020-03-19
+
+last-updated: 2020-03-19
+
+status: 
+
+see-also:
+  - "/enhancements/opm-mirroring.md"  
+
+replaces:
+
+superseded-by:
+---
+
+# OLM Curated Catalog
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions
+- Where is the best place to physically mirror images.  `opm` or `oc`?
```

Comment:
Anything that will manipulate the catalog database file I think will need to be in `opm` and not in `oc`.